libs/pythonparser

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
```

Comment:
Where did you find this script? In the GumTree repo, right?

Could you please try to use [this](https://github.com/JetBrains-Research/pythonparser/blob/master/src/main/python/pythonparser/pythonparser_3.py) one, because I don't remember clearly, but we rewrite it because the initial version had errors

Also, it is not the right way to put the initial file into the project, please set up it automatically, an example can be found [here](https://github.com/elena-lyulina/ast-canonicalization/blob/develop/src/test/kotlin/org/jetbrains/research/transformations/util/ParserSetup.kt) (but you need only direct parser, not invert)
Also, add tests about setting the correct environment for GumTree, something like [this](https://github.com/elena-lyulina/ast-canonicalization/blob/develop/src/test/kotlin/org/jetbrains/research/transformations/PythonSetupTest.kt)

And finally add a new Gradle task that will set up GumTree locally (if you have troubles with it, please contact me)